benchmarks/2022.json

```diff
@@ -866,7 +866,7 @@
     "submissionMethod": "claims",
     "isToppedOut": true,
     "isHighPriority": false,
-    "isToppedOutByProgram": true,
+    "isToppedOutByProgram": false,
```

Comment:
When I ran the `build:benchmarks` script, it also made this update. Looking at the original csv file, it appears this update is correct. @michelingle can you confirm? This is measure id `117` with submission method `claims`

Reply:
Reverted this change and added https://jira.cms.gov/browse/QPPA-6274 for this issue